src/golang/cmd/server/middleware/authentication/authentication.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"net/http"
 
-	"github.com/aqueducthq/aqueduct/cmd/server/utils"
+	"github.com/aqueducthq/aqueduct/cmd/server/response"
+	"github.com/aqueducthq/aqueduct/cmd/server/routes"
 	"github.com/aqueducthq/aqueduct/lib/collections/user"
+	aq_context "github.com/aqueducthq/aqueduct/lib/context"
```

Comment:
The PR description says this is named `context_parser` btw. Having to alias this import everywhere may be a bit annoying - what about "request_context"?